interface/patient_file/encounter/Services/ViewHelper.php

```diff
@@ -0,0 +1,264 @@
+<?php
+/**
+ * Helper functions for an encounter view files
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License as published by the Free Software Foundation; either version 3 of the License, or (at your option)
+ * any later version. This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+ * License for more details. You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @subpackage Encounter
+ * @license GNU GPL3
```

Comment:
The wordage of the license ("of the License, or (at your option) any later version.") allows us to seamlessly go to next license without changing the headers (ie, even if it states 2 or 3, it can still be considered 4); ie. it's basically on open door.